src/main.rs

```diff
@@ -1,394 +1,32 @@
 use std::fs::File;
```

Comment:
I agree here that separating functionality out into a library module is probably best practice here, but this shouldn't be part of the Mayhem integration PR. Please make a PR to the upstream directly with your recommended changes, and only use this PR to integrate Mayhem.

Reply:
I changed the PR to only be mayhem integration. Since the project is a binary by default, I just set up a simple mayhemfile without libfuzzer harnesses